src/PgManyToManyRelationEdgeTablePlugin.js

```diff
@@ -0,0 +1,211 @@
+module.exports = function PgManyToManyEdgeTablePlugin(
+  builder,
+  { pgSimpleCollections }
+) {
+  builder.hook("GraphQLObjectType:fields", (fields, build, context) => {
+    const {
+      extend,
+      getTypeByName,
+      pgGetGqlTypeByTypeIdAndModifier,
+      graphql: { GraphQLNonNull, GraphQLList },
+      inflection,
+      getSafeAliasFromResolveInfo,
+      getSafeAliasFromAlias,
+      pgQueryFromResolveData: queryFromResolveData,
+      pgAddStartEndCursor: addStartEndCursor,
+      pgSql: sql,
+      describePgEntity,
+    } = build;
+    const {
+      scope: { isPgManyToManyEdgeType, pgManyToManyRelationship },
+      fieldWithHooks,
+      Self,
+    } = context;
+    if (!isPgManyToManyEdgeType || !pgManyToManyRelationship) {
+      return fields;
+    }
+
+    const {
+      leftKeyAttributes,
+      junctionLeftKeyAttributes,
+      rightTable,
+      rightKeyAttributes,
+      junctionRightKeyAttributes,
+      junctionTable,
+      junctionRightConstraint,
+      allowsMultipleEdgesToNode,
+    } = pgManyToManyRelationship;
+
+    if (!allowsMultipleEdgesToNode) {
+      return fields;
+    }
+
+    const JunctionTableType = pgGetGqlTypeByTypeIdAndModifier(
+      junctionTable.type.id,
+      null
+    );
+    if (!JunctionTableType) {
+      throw new Error(
+        `Could not determine type for table with id ${junctionTable.type.id}`
+      );
+    }
+    const JunctionTableConnectionType = getTypeByName(
+      inflection.connection(JunctionTableType.name)
+    );
```

Comment:
```suggestion
    );
    if (!JunctionTableConnectionType) { return fields; }
```

Reply:
I feared that might introduce unexpected behavior, for example with `simpleCollections: "only"` or other cases where that Connection type might not exist, so I took a slightly different approach: https://github.com/graphile-contrib/pg-many-to-many/pull/19/commits/8f7b3296d79f3cc85b7379355e1540f582371bd3